insights/ui/search.py

```diff
@@ -0,0 +1,19 @@
+from locators import insight_page_locator
+from selenium_utility import SeleniumUtility
+from navigator import Navigator
+
+
+class UISearch(SeleniumUtility):
+    def register_system(self, hostname):
+        # Navigate to Insight_tab
+        current_url = self.browser.current_url
+        self.browser.get(current_url + 'insights/actions')
+        self.browser.implicitly_wait(30)
+        # Click on system tab
+        nav = Navigator(self.browser)
```

Comment:
object creation should go in __init__ method of the same class so that you can use nav object for other methods as well going forward.